packages/devtools_app/lib/src/utils.dart

```diff
@@ -67,6 +67,15 @@ String percent2(double d) => '${(d * 100).toStringAsFixed(2)}%';
 final NumberFormat _kbPattern = NumberFormat.decimalPattern()
   ..maximumFractionDigits = 0;
 
+String prettyPrintBytes(int size) {
+  final sizeInKB = size / 1024.0;
+  if (sizeInKB < 1024.0) {
+    return '[${printKb(size)} KB]';
```

Comment:
the `[` and `]` shouldn't be in this method as that is a detail of how you want to display the # of byes in a larger string not part of how you want the bye string formatted.
Also, showing the KB and MB units is something that should be an option for the printKb and printMb helpers. In fact is is a bit odd we are displaying some sizes without a unit. For example, add a named parameter `includeUnit` to those methods..

Reply:
Done. I noticed that there are unit tests in place for `printMb` and `printKb` methods. Should there be some for `prettyPrintBytes` as well?

Reply:
yes it would be a good idea to write a quick unittest for this. you should also write a golden image test for the treemap code verifying that it renders as expected given some canned data.

Reply:
Unit test done. Writing golden test now

Reply:
I've been digging around the tests folder to learn more about the structure that's already in place. It seems like there aren't any tests written for the `HeapTree` widget which includes the bottom half of the memory screen. I was thinking it might best to add more regular tests in addition to a golden image test, but wasn't sure if all of the testing should be included in this PR or a separate PR.